setup.sh

```diff
@@ -13,6 +13,26 @@ exec 3>&-
 
 sudo mv jarvis /usr/local/bin/jarvis
 
+# Fedora based (>=22)
+if [[ -f "/etc/dnf/dnf.conf" ]]; then
+  sudo dnf install -y ffmpeg python-pip python-dbus notify-python
+  # chromedriver-install
+  wget https://chromedriver.storage.googleapis.com/2.32/chromedriver_linux64.zip
+  unzip chromedriver_linux64_2.3.zip
+  sudo cp chromedriver /usr/bin/chromedriver
+  sudo chown root /usr/bin/chromedriver
+  sudo chmod 755 /usr/bin/chromedriver
+
+# Debian based
+elif [[ -f "/etc/apt/sources.list" ]]; then
+  sudo apt-get install ffmpeg python-pip python-imdbpy python-notify2
+  sudo apt-get install python-dbus python-dbus-dev libssl-dev libffi-dev libdbus-1-dev libdbus-glib-1-dev
+  sudo apt-get install chromium-chromedriver python2.7
+
+# Arch based
+elif [[ -f "/etc/pacman.conf" ]]; then
+  sudo pacman -S --noconfirm ffmpeg openssl libffi python-pip
```

Comment:
![screenshot from 2017-10-28 10-36-00](https://user-images.githubusercontent.com/32510534/32131460-25fd1b96-bbcc-11e7-8d29-9310aa32f4f0.png)

Reply:
@lucaspbordignon Please confirm that python-pip works for Python 2